src/components/Stepper/Step.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { forwardRef, useContext } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { isNotUndefined } from 'ramda-adjunct';
+
+import { getColor, getSpace, pxToRem } from '../../utils';
+import { SpaceSizes } from '../../theme';
+import { Text } from '../typography/Text';
+import { TextSizes, TextVariants } from '../typography/Text/Text.enums';
+import { Inline, Padbox, Stack } from '../layout';
+import { StepProps } from './Stepper.types';
+import { StepperContext } from './Stepper.context';
+import { StepperOrientations } from './Stepper.enums';
+import StepBullet, { BulletCircle, bulletSize } from './StepBullet';
+
+const StepLabel = styled(Text)`
+  max-width: 26ch;
+`;
+
+const StepSummary = styled(Text)`
+  max-width: 30ch;
+`;
+
+const StepButton = styled.button`
+  border: 0 none;
+  background: transparent none;
```

Comment:
Shouldn't be just one?

Reply:
well yes and no :D it would work in 99 % of cases but :
```
border: 0 === border-width: 0
border: none === border-style: solid
background: transparent === background-color: transparent
background: none === background-image: none
```
TBH inheritance is a bitch 🤣